src/modules/drive/Toolbar/components/ViewSwitcher.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import ToggleButton from 'cozy-ui/transpiled/react/ToggleButton'
+import ToggleButtonGroup from 'cozy-ui/transpiled/react/ToggleButtonGroup'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import ListMinIcon from 'cozy-ui/transpiled/react/Icons/ListMin'
+import MosaicMinIcon from 'cozy-ui/transpiled/react/Icons/MosaicMin'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+/**
+ * ViewSwitcher component for toggling between grid and list views
+ * @param {Object} props - Component props
+ * @param {boolean} props.isBigThumbnail - Whether the thumbnail size is big (grid view)
+ * @param {Function} props.toggleThumbnailSize - Function to toggle the thumbnail size
+ * @param {string} props.className - Additional CSS class name
+ * @returns {JSX.Element} The rendered component
+ */
+const ViewSwitcher = ({ isBigThumbnail, toggleThumbnailSize, className }) => {
```

Comment:
we can rely on the hook provider to get `isBigThumbnail, toggleThumbnailSize` instead of getting them by props. Unless you had a good reason to do this way? 🤔

Reply:
You done it 👍